### PR TITLE
Add in the sendInvite API endpoint

### PR DESCRIPTION
**Pull request by BarryCarlyon**
Working on customer stuff at $dayjob again

Testing with activations and spotted that the sendInvite API call is missing from the module. Added the call

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -236,6 +236,7 @@ shopify.metafield.create({
   - `update(id, params)`
 - customer
   - `accountActivationUrl(id)`
+  - `sendInvite(id[, params])`
```
Comment (lpinca) [inline README.md]: Please keep it sorted.

File: resources/customer.js [hunk 2]
```diff
@@ -46,4 +46,17 @@ Customer.prototype.accountActivationUrl = function accountActivationUrl(id) {
...
+ * Generates a invite email for a customer.
+ *
+ * @param {Number} id Customer ID
+ * @param {Object} params Optional params for adjusting the sent email
+ * @return {Promise} Promise that resolves with the result
+ * @public
+ */
+Customer.prototype.sendInvite = function sendInvite(id, params) {
+  const url = this.buildUrl(`${id}/send_invite`);
+  return this.shopify.request(url, 'POST', 'customer_invite', params).then(body => body.customer_invite);
```
Comment (lpinca) [inline resources/customer.js]: This should be

```js
return this.shopify.request(url, 'POST', 'customer_invite', params || {});
```

or it won't work with default invite.
The `then()` part is wrong as the promise is already resolved with ` body[key]` if the `key` argument is used in `Shopify.prototype.request()`.

**Comment by lpinca** [comment]
We also need tests.

**Comment by BarryCarlyon** [comment]
Derp forgot the tests will take a look on Monday…

**Comment by lpinca** [comment]
Thanks.

**Pull request merged**